src/components/manage/Blocks/Block/Edit.jsx

```diff
@@ -82,7 +91,7 @@ class Edit extends Component {
     const tab = this.props.manage
       ? 1
       : blocks.blocksConfig?.[type]?.sidebarTab || 0;
-    if (this.props.selected) {
+    if (this.props.selected && this.props.editable) {
```

Comment:
@giuliaghisini I guess it's here... in a normal edit or add form, `this.props.editable` it should also be true, right?

Reply:
yes!